### PR TITLE
Update dashboard.md

### PR DESCRIPTION
**Pull request by zwtan**
Step 3 process for rebooting gateway should be in gateway process step. During this period, gateway is not installed yet

### CODE REVIEW

**Comment by marksou** [comment]
@joshblakeley can you confirm @zwtan changes?

File: tyk-docs/content/get-started/with-tyk-on-premise/installation/redhat-rhel-centos/dashboard.md [hunk 1]
```diff
@@ -195,12 +195,11 @@ That's it, your Dashboard is now ready to be bootstrapped.
 
 If all is going well, you will be taken to a log in screen - we'll get to that soon.
 
-### Step 3: Restart the dashboard and start the gateway process
+### Step 3: Restart the dashboard
 
 Because we've just entered a license via the UI, we need to make sure that these changes get picked up, so to make sure things run smoothly, we restart the dashboard process (you only need to do this once) and then start the gateway:
```
Comment (joshblakeley) [inline tyk-docs/content/get-started/with-tyk-on-premise/installation/redhat-rhel-centos/dashboard.md]: Remove the "and then start the gateway" from line 200. Then it looks good from there.

**Review by joshblakeley — CHANGES_REQUESTED** [review]
See comment

**Comment by zwtan** [comment]
Okay updated w/ the removed 'and then start the gateway' from line 200

**Pull request merged**